README.md

```diff
@@ -1,4 +1,4 @@
-# Tanks of Freedom
+# Tanks of Freedom - Added Spanish translation
```

Comment:
This is the title of the readme file. Do not change it's title like that.

Reply:
I reverted the change